### PR TITLE
Kotlin 1.4.10, ktor 1.4.0, KXS 1.0

### PR DESCRIPTION
**Pull request by Denire**
This shifts Kotlin version to 1.4.10, also shifting Ktor to 1.4.0 and kotlinx.serialization to 1.0
Also upgrades _kotlin-telegram-bot_ library for **telegram** channel to 6.0.1 version (as it supports 1.4.10 kotlin).

### CODE REVIEW

File: channels/telegram/src/main/kotlin/com/justai/jaicf/channel/telegram/TelegramReactions.kt [hunk 1]
```diff
@@ -177,7 +178,16 @@ class TelegramReactions(
         replyToMessageId: Long? = null,
         replyMarkup: ReplyMarkup? = null
     ) = api.sendVenue(
-            chatId, latitude, longitude, title, address, foursquareId, foursquareType, disableNotification, replyToMessageId, replyMarkup
```
Comment (morfeusys) [inline channels/telegram/src/main/kotlin/com/justai/jaicf/channel/telegram/TelegramReactions.kt:180]: Please rollback a single line to uniform with other methods

**Review by morfeusys — APPROVED** [review]

File: buildSrc/src/main/kotlin/Version.kt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
     const val slf4j = "1.7.30"
     const val jUnit = "5.6.0"
     const val jetty = "9.4.3.v20170317"
 
-    const val ktor = "1.3.1"
-    const val serializationRuntime = "0.14.0"
-    const val coroutinesCore = "1.3.3"
+    const val ktor = "1.4.0"
+    const val serializationRuntime = "1.0.0"
+    const val coroutinesCore = "1.3.9"
```
Comment (nikvoloshin) [inline buildSrc/src/main/kotlin/Version.kt:14]: Why not 1.4.0+?
There are breaking changes in 1.4.0, it's easier to pass through it now, while we don't have a big coroutines' codebase

Comment (Denire) [reply buildSrc/src/main/kotlin/Version.kt:14]: Ktor dependencies use coroutines 1.3.9, omit 1.3.9 to 1.4.1 in runtime may bring troubles.

File: channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt [hunk 3]
```diff
@@ -32,15 +30,16 @@ open class JaicpReactions : Reactions() {
                 ""
             }
         }
-
-        return json {
+        val obj = buildJsonObject {
```
Comment (nikvoloshin) [inline channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt:33]: Why not to inline this variable?

File: channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt [hunk 4]
```diff
@@ -32,15 +30,16 @@ open class JaicpReactions : Reactions() {
...
             if (this@JaicpReactions is TelephonyReactions) {
-                "dialer" to dialer.getApiResponse()
+                put("dialer", dialer.getApiResponse())
             }
-            "replies" to jsonArray {
-                jsonReplies.forEach { +it }
-            }
-            "answer" to answer
+            put("replies", buildJsonArray {
```
Comment (nikvoloshin) [inline channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/JaicpReactions.kt:37]: There are ready to use `putJsonArray` and `putJsonObject` methods, that do exactly the same

File: channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/ProcessCompatible.kt [hunk 5]
```diff
@@ -4,29 +4,29 @@ import com.justai.jaicf.channel.http.asHttpBotRequest
...
     return JaicpBotResponse.fromRequest(botRequest, response, processingTime)
 }
 
-private fun addRawReply(rawResponse: JsonElement) = json {
-    "replies" to jsonArray {
-        +(json {
-            "type" to "raw"
-            "body" to rawResponse
+private fun addRawReply(rawResponse: JsonElement) = buildJsonObject {
+    put("replies", buildJsonArray {
```
Comment (nikvoloshin) [inline channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/ProcessCompatible.kt:25]: There are ready to use `putJsonArray` and `putJsonObject` methods, that do exactly the same

**Review by nikvoloshin — APPROVED** [review]

**Pull request merged**